os_stub/openssllib/openssl_gen/prov/ciphercommon.h

```diff
@@ -42,6 +42,13 @@ typedef int (PROV_CIPHER_HW_FN)(PROV_CIPHER_CTX *dat, unsigned char *out,
 #define PROV_CIPHER_FLAG_INVERSE_CIPHER   0x0200
 
 struct prov_cipher_ctx_st {
+    /* place buffer at the beginning for memory alignment */

```

Comment:
This file was generated by OpenSSL, correct?

Reply:
Yes, this file is same with
`openssl\providers\implementations\include\prov\ciphercommon.h`